src/assets/scss/theme.scss

```diff
@@ -59,6 +59,10 @@ $app-green: #2e8540;
 $border: #899eb2;
 $gold: $BCgovGold5;
 
+// Colours that should use standard palette instead:
+$other-bg: #e2e8ee;
+$caution: #bf4803;
```

Comment:
Since these should use other colours instead, can you please check with Scott what these should be?

Reply:
Will update once I have response from Scott